docs/reference/esql/processing-commands/sort.asciidoc

```diff
@@ -1,35 +1,56 @@
 [discrete]
 [[esql-sort]]
 === `SORT`
-Use the `SORT` command to sort rows on one or more fields:
+
+**Syntax**
+
+[source,esql]
+----
+SORT column1 [ASC/DESC][NULLS FIRST/NULLS LAST][, ..., columnN [ASC/DESC][NULLS FIRST/NULLS LAST]]
+----
+
+*Parameters*
+
+`columnX`::
+The column to sort on.
+
+*Description*
```

Comment:
Since the original version of the documentation for `sort`, there has been an [improvement](https://github.com/elastic/elasticsearch/pull/98337) made to this command: If a column that is being sorted on has multi-values, then if the ordering is `asc` we will use the `min` value from a multi-value row. If the ordering is `desc` we will use the `max` value from a multi-value row.

Reply:
Similar to what ES does but with only two non-modifiable options: https://www.elastic.co/guide/en/elasticsearch/reference/current/sort-search-results.html#_sort_mode_option

Reply:
Thanks! I've added the following sentence: _Sorting on multivalued columns uses the lowest value when sorting ascending and the highest value when sorting descending._